medical_prescription/models/medical_prescription_order_line.py

```diff
@@ -43,3 +43,8 @@ def name_search(self, name='', args=None, operator='ilike', limit=100):
 
         recs = self.search(domain + args, limit=limit)
         return recs.name_get()
+
+    @api.onchange('medical_medication_id')
+    def _onchange_medical_medication_id(self):
+        self.dispense_uom_id = \
```

Comment:
This line is untested.

Note about onchanges, they are triggered by a view action & not automatically triggered if you are in ORM. When you make the change in ORM that would normally trigger the onchange, you'll need to call the method manually instead. 

Might also want to install the [CodeCov browser plugin](https://github.com/codecov/browser-extension), which makes spotting hot spots much easier (note the red):

![image](https://cloud.githubusercontent.com/assets/7397578/22342969/1b786508-e3ab-11e6-9619-bcc250e6a551.png)